libyul/backends/wasm/EWasmCodeTransform.cpp

```diff
@@ -65,6 +66,8 @@ wasm::Expression EWasmCodeTransform::generateMultiAssignment(
 	if (_variableNames.size() == 1)
 		return { std::move(assignment) };
 
+	allocateGlobals(_variableNames.size() - 1);
```

Comment:
Why was this only introduced now?

Reply:
I guess because we don't have any tests ;)